### ISSUE TITLE
Full Site

### ISSUE THREAD

**Issue by loklaan** [campjs/campjs-x#5]
https://campjs.org
https://x.campjs.org

---

## Tasks

**Generally, designing & developing a full site to accompany "pre" and "post" release of the event's schedule.**

#### Requirements:
* Should be a11y compliant
* Visual experiences should be progressive-enhancements (eg. Landing page)
* Offline-first (SW caching etc)
* Sitemap:
    - Landing
    - Call for papers
    - About
    - Code of conduct
    - Tickets
    - Diversity
    - **post-schedule**: Talks / speakers
    - **post-schedule**: Schedule
    - **on-the-day**: "Things to Click"

#### Nice to haves:
* Embedded ticket purchase
* Embedded CFP form
* Personalised schedule (eg. bookmarking)
* Feedback for speakers (eg. assisted feedback against schedule)
* Chat app (eg. iframe with scuttlebut...?)

#### Infrastructure:

* Statically generated site 100%
* It'd be swell if we could figure out the intranet-first deployment strategy before the event
* Web deployments are through netlify
* Offline-first pushes our content management strategy towards markdown (or mdx), but that's ok

 _Partnering with [Liana Modolo](https://www.linkedin.com/in/lianamodolo/)_
